eng/pipelines/templates/steps/analyze.yml

```diff
@@ -18,7 +18,10 @@ steps:
       workingDirectory: $(Pipeline.Workspace)
     displayName: Dump Package properties
     condition: and(succeeded(), ${{ parameters.IsSdkLibrary }})
-
+  - template: /eng/common/pipelines/templates/steps/publish-artifact.yml
```

Comment:
@benbp is the analyze job the best place for this? Should it instead go in the Build job? 

@raych1 if this stays here then what happens for anyone that sets isSdkLibrary false? they won't have any PackgeInfo files produced.

Reply:
If IsSdkLibrary is set to false then this publish artifact step will fail as there will not be any files to publish. We might need to add the same condition on the publish step.

Reply:
Good catch, I added the same condition.